File: app/src/main/java/kagura/project/com/a8/objects/Card.java
package kagura.project.com.a8.objects;

import android.view.View;

public class Card {

    public View viewFront;
    public View viewBack;
    public View view;
    public int position;

    public Card(View viewFront, View viewBack, int position) {
        this.viewFront = viewFront;
        this.viewBack = viewBack;
        this.position = position;
    }

    public Card(View view, int position){
        this.view = view;
        this.position = position;
    }
}

File: app/src/main/java/kagura/project/com/fruitlegumes/collections/Card.kt
package kagura.project.com.fruitlegumes.collections

import android.view.View

class Card {
    lateinit var viewFront: View
    lateinit var viewBack: View
    lateinit var view: View
    var position: Int

    constructor(viewFront: View, viewBack: View, position: Int) {
        this.viewFront = viewFront
        this.viewBack = viewBack
        this.position = position
    }

    constructor(view: View, position: Int) {
        this.view = view
        this.position = position
    }
}